docs/application/tizen-studio/native-tools/address-sanitizer.md

```diff
@@ -20,7 +20,7 @@ The Address Sanitizer tool can detect the following types of bugs:
 
 To use the Address Sanitizer:
 
-1. In the **Project Explorer** view, right-click the project and select **Profile As > Profile With Address Sanitizer**.
+1. In the **Project Explorer** view, right-click the project and select **Profile As > Profile With Memory Errors**.
 
    The profiling instruments the code and compiles the project for the Address Sanitizer. Finally, it launches the application on the connected target or emulator.
 
```

Comment:
I think the image needs to be changed. 
![SharedScreenshot](https://user-images.githubusercontent.com/43839737/120167427-5a4f8b00-c21b-11eb-95b6-b5f284cf002f.jpg)

Reply:
https://docs1.stg.tizen.org/staging/1447/application/tizen-studio/native-tools/address-sanitizer/
The latest change seems to be incorrect even if we change the image. Please verify. 

Existing document: 
https://docs.tizen.org/application/tizen-studio/native-tools/address-sanitizer/

Reply:
Nice catch! let me update.
Thank you!